analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -47,15 +47,21 @@ select_metadata <- metadata %>%
 stranded_expression <-
   read_rds(
     file.path(
-      data_dir,
+      root_dir,
```

Comment:
Here's another instance of what I mentioned previously about `data_dir`.

Reply:
This seems like the same as above : since the relative analyses/collapse-rnaseq/results/  would have the updated files from https://github.com/kgaonkar6/OpenPBTA-analysis/blob/all_subtyping_gsea_rerun/scripts/run-for-subtyping.sh I pointed to that folder instead of data_dir.